stable/concourse/values.yaml

```diff
@@ -65,6 +65,19 @@ concourse:
     ##
     enableGlobalResources: true
 
+    ## The Future™ of resource checking.
+    ##
+    enableLidar: false
+
+    ## Interval on which the resource scanner will run to see if new checks need
+    ## to be scheduled.
+    ##
+    lidarScannerInterval: 1m
```

Comment:
Hey, 

I know that in various places we have values set in the yaml, but I'd rather not have these here - with an empty value, we'd then default to whatever we set as default in Concourse, which gives us less trouble if we end up changing defaults in Concourse itself 🤔  

Would you mind changing these to be empty?

Thanks!

Reply:
done